src/main/java/spoon/reflect/meta/impl/RoleHandlerHelper.java

```diff
@@ -71,4 +78,51 @@ public static RoleHandler getOptionalRoleHandler(Class<? extends CtElement> targ
 		}
 		return null;
 	}
+
+	/**
+	 * @param targetClass a Close whose handlers we are looking for
```

Comment:
I don't understand you're documentation here: what's a "Close"?

Reply:
I should sleep more ... at least sometime ... "Close" is wrong. Correct is "Class" ;-) I will fix it.